pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/minion/PinotHelixTaskResourceManager.java

```diff
@@ -452,6 +640,25 @@ public void addToError(int error) {
       _error += error;
     }
 
+    public void addToCompleted(int completed) {
```

Comment:
We can remove all these `addTo*` and only allow `addTaskState`

Reply:
removed the addTo* and only setter is addTaskState.